integration-tests/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
```

Comment:
Missing license header

Reply:
We are using Log4J2. log4j.properties only support Log4J 1.x versions. We need to use log4j2.xml

Reply:
We are using Log4J2. log4j.properties file only supports Log4j 1.x versions. We should be using log4j2.xml

Reply:
You can parse log4j2 configs as a property file too. Since we are using XML version of it, I'll do the changes. Thanks